build.gradle

```diff
@@ -9,19 +9,38 @@ repositories {
 }
 
 sourceSets {
-    integrationTest {
+    integrationTestCassandra {
         java {
             compileClasspath += main.output + test.output
             runtimeClasspath += main.output + test.output
             srcDir file('src/integration-test/java')
+            exclude '**/com/scalar/db/storage/cosmos/*.java'
```

Comment:
I'm sure you have tried it but I'm wondering if there is any way to just include what to test or exclude except what to test. Let's chat.

Reply:
As discussed, we can use `include` here which is better than `exclude`. I will push a new commit for it.